usr/share/rear/prep/GNU/Linux/280_include_virtualbox.sh

```diff
@@ -0,0 +1,11 @@
+# Pick up virtualbox modules and tools.
+if lsmod | grep -q vbox ; then
+    # virtualbox modules (if present)
+    VBOX_MODULES=( vboxsf vboxvideo vboxguest )
+    MODULES=( ${MODULES[@]} ${VBOX_MODULES[@]} )
+    CLONE_USERS=( ${CLONE_USERS[@]}  vboxadd )
+    CLONE_GROUPS=( ${CLONE_GROUPS[@]} vboxusers )
+    VBOX_COPY_AS_IS=( /etc/init.d/vboxadd* /opt/VBoxGuestAdditions-* /usr/sbin/VBoxService )
+    COPY_AS_IS=( ${COPY_AS_IS[@]} ${VBOX_COPY_AS_IS[@]} )
+    Log "Adding virtualbox modules"
+fi
```

Comment:
GitHub shows here a "no newline at end of file" warning icon.
To be safe better add one (I remember really weird behaviour
in special cases with ASCII files without a newline at the end).

Reply:
Thanks!